src/nupic/algorithms/SpatialPooler.hpp

```diff
@@ -31,8 +31,10 @@
 #include <vector>
 #include <nupic/math/SparseBinaryMatrix.hpp>
 #include <nupic/math/SparseMatrix.hpp>
```

Comment:
these *Matrix can now be removed from SP includes, in separate PR then rm from codebase #104